.github/workflows/cicd.yml

```diff
@@ -21,6 +21,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: psf/black@stable
+        with:
+          options: "--check --verbose"
+          src: "./deploy ./scripts ./lambdas"
+          version: "22.8.0"
```

Comment:
reviewer context: making sure our CI runner is using same version of `psf/black` that our `poetry.lock` specifies and now that we also use locally in pre-commit hook